lib/std/c/haiku.zig

```diff
@@ -702,7 +702,7 @@ pub const T = struct {
     pub const CSETAF = 0x8002;
     pub const CSETAW = 0x8003;
     pub const CWAITEVENT = 0x8004;
-    pub const CSBRK = 08005;
+    pub const CSBRK = 0x8005;
```

Comment:
This brings me back.  I'll share the story for those interested. Years ago at my old job we were sometimes getting weird file permissions.  It was a very large code base (millions of LOC in hundreds of thousands of files) and was hard to reproduce.  Various people had spent days trying to reproduce/debug this issue but it's difficult when there's so much code to sift through.  I thought it could be some sort of uninitialized value issue but I was able to determine that each time it happened the permission bits were consistent.  After looking at the bit pattern of the permissions, it dawned on me that a user probably entered a hex integer literal when it was meant to be octal (i.e. they did something like `0x755` instead of `0755`).  I took the bit pattern and decoded it into a hex integer literal and used grep to search our repositories (probably took 5 or 10 minutes) and sure enough, I got a hit for a set of permission bits being passed to an `open` call! Changed that `0x` to an `0` and problem solved.

Reply:
Wow, thanks for sharing! That really sounds like a pain. So much time spent for such a simple fix. Luckily with Zig this should never happen again!

Reply:
I had exact this issue myself a couple of weeks ago in my zig codebase, and thought about suggesting this just a few days ago. @r00ster91 you got it , thanks.